Makefile

```diff
@@ -124,6 +126,27 @@ else
 	cmake --build ./BuildTests $(VERBOSE) --target test
 endif
 
+test-cpp-mpi:
+	rm -rf ./BuildTests
+	cmake -BBuildTests -G Ninja \
+		  -DCMAKE_BUILD_TYPE=Debug \
+		  -DBUILD_TESTS=ON \
+		  -DENABLE_WARNINGS=ON \
+		  -DPL_BACKEND=lightning_gpu \
+		  -DENABLE_MPI=ON \
+		  $(OPTIONS)
+ifdef target
+	cmake --build ./BuildTests $(VERBOSE) --target $(target)
+	mpirun -np 2 ./BuildTests/$(target)
```

Comment:
The purpose of this addition is only to run the MPI testing suite in a simple way because we do not have, as far as I know, instructions on how to run the MPI test.

Reply:
Sounds good!